docs/local-development.md

```diff
@@ -4,9 +4,7 @@ Local development environment for exercising Tekton and Sigstore running on kind
 
 **Note**: We have to use `--allow-insecure-registry` due to this [cosign bug](https://github.com/sigstore/cosign/issues/1405).
 
-## Notes for macOS
-
-You may hit file limits; you can run `sudo launchctl limit maxfiles 65536 200000` to remediate that issue.
+## Notes for MacOS
 
 The airplay receiver uses port 5000, which may need to be disabled. Further details via [Apple's developer forum](https://developer.apple.com/forums/thread/682332). Alternatively, you can manually modify the script and change the [REGISTRY_PORT](https://github.com/vaikas/sigstore-scaffolding/blob/main/hack/setup-mac-kind.sh#L19)
```

Comment:
In Prereqs can add the following Docker requirements:

5 CPUs

2GB Memory

1 GB Swap

Reply:
Yeah, I put it here, but I'll reformat it :)
https://github.com/Cray-HPE/sigstore-poc/pull/17/files#diff-53e532051b05508d4e9b669b7dbb53f2d317faf2d91cb965e0bf98cc1a2133deR29